bundles/framework/layerlist/service/layerlist.js

```diff
@@ -0,0 +1,60 @@
+(function (Oskari) {
+    Oskari.clazz.define('Oskari.mapframework.service.LayerlistService',
+        function () {
+            this.layerlistFilterButtons = {};
+            Oskari.makeObservable(this);
+        },
+        {
+            /** @static @property __name service name */
+            __name: 'LayerlistService',
+            /** @static @property __qname fully qualified name for service */
+            __qname: 'Oskari.mapframework.service.LayerlistService',
+            /**
+             * @method getQName
+             * @return {String} fully qualified name for service
+             */
+            getQName: function () {
+                return this.__qname;
+            },
+            /**
+             * @method getName
+             * @return {String} service name
+             */
+            getName: function () {
+                return this.__name;
+            },
+            getMutator () {
```

Comment:
getMutator() is something that has been left from a previous version as this.mutator isn't initialized?

Reply:
Refactored singleton layerlist service in layerselector2 and removed this file.